### PR TITLE
Feature/tech debt 1

### PR DESCRIPTION
**Pull request by MichaelRobinson-BJSS**

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Marketing/Views/Solution/Index.cshtml [hunk 1]
```diff
@@ -331,7 +331,7 @@
             <div data-test-id="dashboard-sections-box" class="bc-c-section-box-border nhsuk-u-padding-4 nhsuk-u-margin-bottom-5">
                 <h3>Sections added by NHS Digital</h3>
                 <p>The following sections of your Catalogue Solution page will be completed for you by NHS Digital. The information will be taken from your Catalogue Solution’s framework tender and capabilities assessment.</p>
-                <div data-test-id="" class="bc-c-data-bulletlist ">
+                <div data-test-id="">
```
Comment (bjss-chris) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Marketing/Views/Solution/Index.cshtml:334]: I think we may as well remove the `data-test-id` attribute as it has no value. Unless it should have one, @PhysicsJon?

**Review by bjss-chris — COMMENTED** [review]
Looks good – just one comment/question.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Marketing/Views/Solution/Index.cshtml (hunk 1 above)
Comment (MichaelRobinson-BJSS) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Marketing/Views/Solution/Index.cshtml]: Didn't impact any tests by removing it

Comment (bjss-chris) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Marketing/Views/Solution/Index.cshtml]: On a call with @PhysicsJon and he says we can delete the `data-test-id` attribute here.

**Review by bjss-chris — APPROVED** [review]

**Pull request merged**